server/topic.go

```diff
@@ -306,8 +306,15 @@ func (t *Topic) runLocal(hub *Hub) {
 					log.Printf("topic[%s] subscription failed %v, sid=%s", t.name, err, join.sess.sid)
 				}
 			}
+			if join.sess.inflightReqs != nil {
```

Comment:
How can it be `nil`?

Reply:
Multiplexed topic master (short lived) sessions don't have it yet. I should perhaps add support for these too as they modify the master topic state.